### PR TITLE
Support JSON I/O in CLI

### LINKED ISSUE #67: coherent output between solc and solcjs
Is there a reason why which are output by the native solc compiler differ from the solcjs output?
It would be great to receive the same output from both.

e.g. 
bytecode vs. bin
runtimeBytecode vs. bin-runtime
interface vs. abi

### PR DESCRIPTION
**Pull request by axic**
Fixes #67 and #97.

### CODE REVIEW

File: solcjs [hunk 1]
```diff
@@ -45,6 +48,26 @@ function abort (msg) {
   process.exit(1);
 }
 
+if (argv['standard-json']) {
+  if (!solc.supportsStandard) {
+    abort('Compiler does not support Standard JSON I/O');
+  }
+
+  var size = fs.fstatSync(process.stdin.fd).size;
```
Comment (chriseth) [inline solcjs]: Are you able to do that on a stream? Will it store the data somewhere?

File: solcjs [hunk 2]
```diff
@@ -45,6 +48,26 @@ function abort (msg) {
...
+
+  if (size <= 0) {
+    abort('Empty input was read');
+  }
+
+  var input = fs.readSync(process.stdin.fd, size)[0];
+
+  console.log(solc.compileStandard(input));
```
Comment (chriseth) [inline solcjs]: Shouldn't we use `process.stdout` here?

Comment (axic) [inline solcjs]: `console.log` goes to stdout, `console.error` is stderr

File: solcjs (hunk 1 above)
Comment (axic) [inline solcjs]: Don't know, took this code from stackoverflow and it works (it counts till EOF).

Comment (chriseth) [inline solcjs]: I don't think we should use `process.stdin.fd`: https://github.com/nodejs/node/issues/7439#issuecomment-228760636

File: solcjs (hunk 2 above)
Comment (chriseth) [inline solcjs]: ok!

File: solcjs (hunk 1 above)
Comment (chriseth) [inline solcjs]: Ok, so let's keep that and hope that it works for everyone :-)

**Pull request merged**